src/update_uuids/modify_fileUuids_in_db.py

```diff
@@ -0,0 +1,55 @@
+import json
+import pymongo
+import requests as rq
+import os
+import uuid
+
+
+def find_file_ids(project_uuid):
+    uuid_to_fileuuid = {}
+    azul_url = os.getenv('azul_url') or "https://service.azul.data.humancellatlas.org/index/files"
```

Comment:
1. Use https://docs.python.org/3/library/os.html#os.getenv
2. The convention is that env variables are uppercase
```suggestion
    azul_url = os.getenv('AZUL_URL', default= "https://service.azul.data.humancellatlas.org/index/files")
```